Oct 26 23:44:42 jamesmacbookpro main[72126]: main Error
Oct 26 23:44:42 jamesmacbookpro main[72126]: 2016-10-26 23:44:42.881 main[72126:4588758] main Error
Oct 26 23:44:44 jamesmacbookpro main[72126]: Traceback (most recent call last):
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "/Users/james/projects/spafit/test_py2app/dist/main.app/Contents/Resources/__boot__.py", line 351, in <module>
Oct 26 23:44:44 jamesmacbookpro main[72126]:     _run()
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "/Users/james/projects/spafit/test_py2app/dist/main.app/Contents/Resources/__boot__.py", line 336, in _run
Oct 26 23:44:44 jamesmacbookpro main[72126]:     exec(compile(source, path, 'exec'), globals(), globals())
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "/Users/james/projects/spafit/test_py2app/dist/main.app/Contents/Resources/main.py", line 4, in <module>
Oct 26 23:44:44 jamesmacbookpro main[72126]:     from PyQt5.QtWidgets import QApplication, QLabel
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "PyQt5/QtWidgets.pyc", line 14, in <module>
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "PyQt5/QtWidgets.pyc", line 10, in __load
Oct 26 23:44:44 jamesmacbookpro main[72126]:   File "imp.pyc", line 342, in load_dynamic
Oct 26 23:44:44 jamesmacbookpro main[72126]: ImportError: dlopen(/Users/james/projects/spafit/test_py2app/dist/main.app/Contents/Resources/lib/python3.5/lib-dynload/PyQt5/QtWidgets.so, 2): Library not loaded: @rpath/QtWidgets.framework/Versions/5/QtWidgets
Oct 26 23:44:44 jamesmacbookpro main[72126]:   Referenced from: /Users/james/projects/spafit/test_py2app/dist/main.app/Contents/Resources/lib/python3.5/lib-dynload/PyQt5/QtWidgets.so
Oct 26 23:44:44 jamesmacbookpro main[72126]:   Reason: image not found
